definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-referral-nonprod.json

```diff
@@ -0,0 +1,88 @@
+[
+  {
+    "LiveFrom": "13/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferral",
+    "CaseFieldID": "GeneralReferralReason",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "generalreferral",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "13/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferral",
+    "CaseFieldID": "GeneralApplicationFrom",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "generalreferral",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GeneralReferralReason=\"generalApplicationReferral\"",
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "13/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferral",
+    "CaseFieldID": "GeneralApplicationReferralDate",
```

Comment:
"FieldShowCondition": "GeneralReferralReason=\"generalApplicationReferral\"",
?

Reply:
`generalApplicationReferral` is one of the options (fixed list items) in GeneralReferralReason. When that is selected this field is displayed

Reply:
ok